controllers/apps/utils.go

```diff
@@ -84,3 +86,67 @@ func clientOption(v *model.ObjectVertex) *multicluster.ClientOption {
 	}
 	return multicluster.InControlContext()
 }
+
+func resolveServiceDefaultFields(obj, objCopy *corev1.ServiceSpec) {
```

Comment:
These parameter names are hard to understand, maybe `oldSpec, newSpec`, or something else.

Reply:
fixed